### PR TITLE
bugfix for inconsistent decision block

### PR DESCRIPTION
**Pull request by pratikbhd**
This PR should handle the bug #184 . Running this through the graph generator might help verify much more easily.

### CODE REVIEW

File: delphi/translators/for2py/scripts/genPGM.py [hunk 1]
```diff
@@ -368,6 +369,7 @@ def genPgm(node, state, fnNames):
             fnName=node.name,
             varTypes=localTypes,
         )
+
```
Comment (skdebray) [inline delphi/translators/for2py/scripts/genPGM.py]: delete

File: delphi/translators/for2py/scripts/genPGM.py [hunk 2]
```diff
@@ -590,6 +596,19 @@ def genPgm(node, state, fnNames):
             or elseDefs[var] != startDefs[var]
         ]
 
+        # print(set(startDefs))
```
Comment (skdebray) [inline delphi/translators/for2py/scripts/genPGM.py]: delete commented-out code

**Comment by adarshp** [comment]
Looks like it works:

screen shot 2019-02-21 at 6 48 22 pm

I approve merging this after the commented out code and print statements are removed.

Thanks @pratikbhd !

File: delphi/translators/for2py/scripts/genPGM.py [hunk 3]
```diff
@@ -759,6 +764,7 @@ def genPgm(node, state, fnNames):
     # Name: ('id', 'ctx')
     elif isinstance(node, ast.Name):
         lastDef = getLastDef(node.id, state.lastDefs, state.lastDefDefault)
+        print(state.lastDefs)
```
Comment (skdebray) [inline delphi/translators/for2py/scripts/genPGM.py]: @pratikbhd Is this a debugging print statement, or is this part of the program logic?  If the former, can you delete it?

Comment (pratikbhd) [reply delphi/translators/for2py/scripts/genPGM.py]: Yes, this is a debugging print statement. I seemed to have missed deleting this one. Just deleted.

File: delphi/translators/for2py/scripts/genPGM.py (hunk 2 above)
Comment (pratikbhd) [reply delphi/translators/for2py/scripts/genPGM.py]: Deleted

File: delphi/translators/for2py/scripts/genPGM.py (hunk 3 above)
Comment (skdebray) [reply delphi/translators/for2py/scripts/genPGM.py]: Thanks. :)

**Pull request merged**